addons/mrp/views/mrp_production_views.xml

```diff
@@ -262,6 +264,15 @@
                                         decoration-warning="not is_done and (quantity_done - should_consume_qty &gt; 0.0001)"
                                         attrs="{'column_invisible': [('parent.state', '=', 'draft')], 'readonly': [('show_details_visible', '=', True)]}"/>
                                     <field name="show_details_visible" invisible="1"/>
+                                    <field name="use_create_lots" invisible="1"/>
+                                    <field name="use_existing_lots" invisible="1"/>
+                                    <field name="lot_ids" widget="many2many_tags"
```

Comment:
Don't forget to add the same field in move_byproduct_ids
https://github.com/odoo/odoo/blob/41527531b7b6f727e25a5a68aedb49609fbb1d3a/addons/mrp/views/mrp_production_views.xml#L277